test/e2e/framework/service_util.go

```diff
@@ -93,6 +94,9 @@ const (
 	// resource and time intensive.
 	MaxNodesForEndpointsTests = 3
 
+	// ServiceEndpointsTimeout is the maximum time in which endpoints for the service should be created.
+	ServiceEndpointsTimeout = 2 * time.Minute
```

Comment:
Is this time sufficient and standard to make sure endpoints are generated till this timeout? For testing we are using `agnhost/serve-hostname or pause` pod expecting them to be up and running till that time.